pkg/ucp/frontend/radius/routes.go

```diff
@@ -70,14 +70,17 @@ func (m *Module) Initialize(ctx context.Context) (http.Handler, error) {
 	}
 
 	ctrlOptions := controller.Options{
-		Address:        m.options.Address,
-		PathBase:       m.options.PathBase,
+		Address:        m.options.Config.Server.Address(),
 		DatabaseClient: databaseClient,
+		PathBase:       m.options.Config.Server.PathBase,
 		StatusManager:  m.options.StatusManager,
+
+		KubeClient:   nil, // Unused by Radius module
+		ResourceType: "",  // Set dynamically
```

Comment:
Wouldn't these get set to nil and empty string by default even if we don't initialize them here?

Reply:
Yes. This is documentation. Someone else might come along and think they are fixing a problem.